resources/teamcity/web/keyman-web-release.sh

```diff
@@ -58,9 +58,7 @@ function _zip_and_upload_artifacts() {
 
   builder_echo start "zip and upload artifacts" "Zipping and uploading artifacts"
 
-  cd "${KEYMAN_ROOT}/resources/teamcity/web"
   powershell -NonInteractive -ExecutionPolicy Bypass -File zip-and-upload-artifacts.ps1
```

Comment:
Maybe late comment - 

If the working directory is `${KEYMAN_ROOT}/web`, will the path to the .ps1 script need to be given?